### ISSUE TITLE
Phpactor Language Server must be initialized with a root URI, NULL provided

### ISSUE THREAD

**Issue by leocavalcante** [phpactor/vscode-phpactor#120]
Not sure what I'm missing here.

<img width="1264" alt="Screenshot 2024-05-08 at 18 01 34" src="https://github.com/phpactor/vscode-phpactor/assets/183722/160d043c-2709-4cc1-9a0f-9628ffb814d6">

**Comment by dantleech** [comment]
Does this happen if you open a folder? Does it _always_ happen? Which version of VS code

**Comment by leocavalcante** [comment]
I just create New Text file with Cmd + N and Change it's mode to PHP

VSCode 1.89

<img width="1125" alt="Screenshot 2024-05-08 at 18 10 13" src="https://github.com/phpactor/vscode-phpactor/assets/183722/b40ee636-478c-4f49-9187-cc1c0bfc02a0">

**Comment by dantleech** [comment]
Phactor needs a project root, try opening the folder where you want to create the file

**Comment by leocavalcante** [comment]
Thank you. It is working now, opening a folder with an existing PHP file.

May I ask. How to make Phpactor gain priority over the basic suggestions?

<img width="687" alt="Screenshot 2024-05-08 at 18 35 33" src="https://github.com/phpactor/vscode-phpactor/assets/183722/17e57c60-a39e-471d-98f3-954ce234b258">

I'd like to see `bar()` and `qux()` as the first suggestions, for example.

**Comment by dantleech** [comment]
IIRC you need to disable VS code's default PHP integration, there will be a setting for it somewhere....

**Comment by leocavalcante** [comment]
> IIRC you need to disable VS code's default PHP integration, there will be a setting for it somewhere....

Yeah, I can set `php.suggest.basic` to `false`, but when I do this, global functions like `array_map`, for example, isn't suggested... They should come up from Phpactor?

**Comment by dantleech** [comment]
They should...
